central/sensor/service/pipeline/clustermetrics/pipeline.go

```diff
@@ -68,6 +67,7 @@ func (p *pipelineImpl) Run(
 	clusterMetrics := msg.GetClusterMetrics()
 	p.metricsStore.Set(clusterID, clusterMetrics)
 	p.infoMetric.SetClusterMetrics(clusterID, clusterMetrics)
+	_ = p.usageStore.UpdateUsage(ctx, clusterID, clusterMetrics)
```

Comment:
Do we want to ignore the error here or log it at Info or Warn?

Reply:
I'm not sure logging an error here would be of any use for the reader. That would be just noisy to log it every 5 minutes, if there is a persistent problem with the database.